### ISSUE TITLE
Exfalso prefs throws TypeError due to missing parent parameter

### ISSUE THREAD

**Issue by declension** [quodlibet/quodlibet#3608]
https://sentry.io/organizations/quodlibet/issues/2245284656/?referrer=github_plugin

```
TypeError: __init__() missing 1 required positional argument: 'parent'
  File "quodlibet/qltk/exfalsowindow.py", line 75, in prefs_cb
    window = PreferencesWindow(self)
  File "quodlibet/qltk/exfalsowindow.py", line 277, in __init__
    super().__init__()

TypeError: __init__() missing 1 required positional argument: 'parent'
```

**Comment by declension** [comment]
Sentry issue: [QUODLIBET-PE](https://sentry.io/organizations/quodlibet/issues/2324947128/?referrer=github_plugin)

**Comment by declension** [comment]
Fixed in d96634d95fda69110722bf14716eacf329d63438
